LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright [2021] [Oleksandr Mazyar]
 
```

Comment:
It is instruction and template how licence should be applied to project files.

This place should be not changed.